processing/src/main/java/org/apache/carbondata/processing/store/writer/AbstractFactDataWriter.java

```diff
@@ -252,6 +252,9 @@ private static long getMaxOfBlockAndFileSize(long blockSize, long fileSize) {
     if (remainder > 0) {
       maxSize = maxSize + HDFS_CHECKSUM_LENGTH - remainder;
     }
+    LOGGER.info("The configured block size is " + blockSize + " byte, " +
```

Comment:
@jackylk  Maybe i should extract if .. else part to a method called ConvertByteToReadable, what's your opinion?